typing/parmatch.ml

```diff
@@ -1231,27 +1227,29 @@ let rec list_satisfying_vectors pss qs =
                     ) constrs
                   )
                 in
-                if full_match false constrs then for_constrs () else
-                match p.pat_desc with
+                if full_match false constrs
+                then for_constrs ()
+                else begin match p.pat_desc with
                 | Tpat_construct _ ->
```

Comment:
Please, no.
  
Also, I only reindented these two lines, I didn't format them. So "reverting a non-standard style that trefis used" my ass. :)

Reply:
I will change back -- personally I think that not having enough delimiters is worse than playing tricks with the indentation level, so the new version is better than the previous one.

Reply:
Ah! But here is the good news: you can have both! Delimiters *and* indentation.

Reply:
Well in fact it's possible to keep `begin match` and revert the rest, so I'm doing that.